addons/point_of_sale/data/point_of_sale_data.xml

```diff
@@ -31,7 +31,7 @@
             <field name="taxes_id" eval="[(5,)]"/>
         </record>
 
-        <record id="uom.product_uom_categ_unit" model="uom.category">
+        <record id="uom.product_uom_unit" model="uom.uom">
```

Comment:
I guess a record for `uom.product_uom_pack_6` should be added too to do the same thing?

Reply:
Yes, indeed, and this is accounted for in this compute: https://github.com/odoo/odoo/pull/184131/files/7e45f79d2819529512906619358588155b0fa393#diff-77bd98bfa91c156792de868abf8835486795424e2347e36bbcc2a754d91fa709R11-R14